src/ReadVTK.jl

```diff
@@ -28,7 +28,7 @@ Hold all relevant information about a VTK XML file that has been read in.
 - `filename`: original path to the VTK file that has been read in
 - `xml_file`: object that represents the XML file
 - `file_type`: currently only `"UnstructuredGrid"` or `"ImageData"` are supported
-- `version`: currently only v1.0 is supported
+- `version`: currently only v1.0 and beyond are supported
```

Comment:
```suggestion
- `version`: VTK XML file format version (v1.0 or later)
```
I think there is no useful spec information on this available publicly, so we can just be very neutral about this

Reply:
Yep